examples/webpack.config.local.js

```diff
@@ -45,6 +51,27 @@ function makeLocalDevConfig(EXAMPLE_DIR = LIB_DIR, linkToLuma) {
   // console.warn(JSON.stringify(LUMA_ALIASES, null, 2)); // uncomment to debug config
   // require('fs').writeFileSync('/tmp/ocular.log', JSON.stringify(config, null, 2));
 
+  const MATH_ALIASES = {};
+  if (linkToMath) {
+    const MATH_MODULES = [
```

Comment:
Nit: We could probably auto discover these by listing the modules directory.

Reply:
I feel this way it is more explicit and allows the user to only import some local modules